src/sentry/api/event_search.py

```diff
@@ -394,6 +397,26 @@ def visit_paren_term(self, node, children):
 
         return ParenExpression(children)
 
+    def visit_boolean_filter(self, node, children):
+        (negation, search_key, sep, search_value) = children
+        is_negated = self.is_negated(negation)
+
+        # Numeric and boolean filters overlap on 1 and 0 values.
+        if self.is_numeric_key(search_key.name):
+            return self.visit_numeric_filter(node, (search_key, sep, "=", search_value))
+
+        if search_key.name in self.boolean_keys:
+            if search_value.text in ("yes", "true", "1"):
```

Comment:
Could we add a `.lower()` so that we accept `YES` and `TrUe` as well?

Reply:
That should be added to the regex above. Otherwise `error.handled:Yes` won't be interpreted as a boolean filter at all.

Reply:
Made it case insensitive now.